guide/src/continuous-integration.md

```diff
@@ -124,3 +124,31 @@ deploy: book
 		git commit -m "deployed on $(shell date) by ${USER}" && \
 		git push origin gh-pages
 ```
+
+## Deploying Your Book to GitLab Pages
+Inside your repository's project root, create a file named `.gitlab-ci.yml` with the following contents:
+```yml
+stages:
+    - deploy
+
+pages:
+  stage: deploy
+  image: rust:latest
+  variables:
+    CARGO_HOME: $CI_PROJECT_DIR/cargo
+  before_script:
+    - export PATH="$PATH:$CARGO_HOME/bin"
+    - mdbook --version || cargo install --debug mdbook
```

Comment:
Any particular reason to use `--debug` here?  On large books, it can be very slow (though I guess on CI it's not too important).  It's also quite a bit larger on disk.

FWIW, I usually suggest that CI installs the precompiled binaries (such as [here](https://github.com/rust-lang/rustup/blob/b7e79deea18b78497f91d218ef3051126f7c77a5/.github/workflows/deploy-docs.yaml#L16-L18)), as it can be substantially faster and avoids dealing with caching issues. I realize this is following the style of the other examples, just putting it out there.

Reply:
No, I've removed it.
I tried using an rust:alpine image and just using the package manager but got errors while trying to install.